src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -234,8 +234,9 @@ the MIT License. See LICENSE in the project root for license information. -->
                                                     HasSettingValue="{x:Bind State.Profile.HasFontFace, Mode=OneWay}"
                                                     ClearSettingValue="{x:Bind State.Profile.ClearFontFace}"
                                                     SettingOverrideSource="{x:Bind State.Profile.FontFaceOverrideSource, Mode=OneWay}">
-                                <TextBox Text="{x:Bind State.Profile.FontFace, Mode=TwoWay}"
-                                         Style="{StaticResource TextBoxSettingStyle}"/>
+                                <ComboBox ItemsSource="{x:Bind FontList, Mode=OneWay}"
+                                          SelectedItem="{x:Bind CurrentFontFace, Mode=TwoWay}"

```

Comment:
What happens if `CurrentFontFace` is something that's not an installed font? Like if I set my `fontFace` to `asdf`, what happens then?

Reply:
So, there's a two fold approach (it's more developed in the upcoming commit):
1. We have 2 lists: all installed fonts and monospace fonts. If we find that the current font face is not in the list of monospace fonts, assume that it's in the list of all fonts. So we update the UI to force the user to look at all of the installed fonts.
2. If the user _still_ selected a font that doesn't exist, `CurrentFontFace()` returns `Cascadia Mono`. This _has_ to be valid since it is shipped with Windows Terminal.

Reply:
I'd rather there be an error or warning instead of a whole "it automatically edits the setting if I look at the settings page" :/

Reply:
Huh, I think we might be ok, but let me run through it out loud to double check.
- JSON - invalid font face in settings.json --> show "Cascadia Mono" in SUI:
![warning dialog](https://user-images.githubusercontent.com/11050425/109344167-3ac35380-7823-11eb-8ca3-706c95df6e65.png)
- SUI - limited to autogenerated list

So the only way to get an invalid font is if you modified the JSON. That said, if you then go to SUI, the change won't go into effect until you click on the combo box.

Maybe the behavior should be that we add another "font" called "Invalid Font"? But I feel like this case is fairly rare, and this is more of a request for input validation in SUI. So I'm inclined to say we should punt on this one or throw it in the backlog? Whatcha think?